package.json

```diff
@@ -41,7 +41,10 @@
         "docker:build": "docker build -t gettestedcovid19/webserver:latest -f docker/Dockerfile .",
         "docker:run": "docker run -p 3000:3000 gettestedcovid19/webserver:latest",
         "docker:push": "docker push gettestedcovid19/webserver:latest",
-        "api:start:dev": "nodemon ./src/server/server.js"
+        "api:docker:build": "docker build -t gettestedcovid19/webserver:latest -f docker/APIDockerfile .",
```

Comment:
wanna change webserver:latest to api:latest? I assume we will still have webserver as the other endpoint?

Reply:
yes